docs/src/tutorials/Installation_and_configuration.md

```diff
@@ -0,0 +1,330 @@
+# Getting started with thin-edge.io on a Raspberry Pi 
+
+After following this tutorial you will have an overview of the installation and configuration of thin-edge.io. As an example, a Raspberry Pi is used. This tutorial explains in small steps to reach the goal of sending data to an IoT cloud platform (in this case, Cumulocity IoT) and performing some additional device management tasks.
+
+
+## Introduction
+
+thin-edge.io is an open-source project initiated by Software AG and partners to provide a cloud-agnostic edge framework. It is much more generic than the device management agent, so it can connect to multiple IoT cloud platforms, and it allows flexible logic executed on the device. It is optimized for a very small footprint and high performance.
+
+The Raspberry PI is a relatively simple and cheap device but powerful. Therefore it is ideal for testing and try-outs and some production use cases.
+
+
+##  Prerequisite
+
+To follow this guide, you only need the following:
+- A [Cumulocity]([https://www.arduino.cc/en/software](https://www.softwareag.cloud/site/product/cumulocity-iot.html#/)) Trial tenant. 
+
+- A Raspberry Pi (3 or 4) with Raspian installed, for other boards and OS'es have a look [here](https://link_to_overview_supported_devices)
```

Comment:
This url is broken too.

See the [docs/READM.md](https://github.com/hansb001/thin-edge.io/blob/New_Documentation/docs/README.md)
to generated the doc on your laptop ... checking any markdown errors.

Reply:
fixed